Makefile.d/dependencies.mk

```diff
@@ -172,3 +172,21 @@ update/vald:
 ## update vald client library made by clojure self version
 update/valdcli:
 	curl --silent https://api.github.com/repos/vdaas/vald-client-clj/releases/latest | grep -Po '"tag_name": "\K.*?(?=")' > $(ROOTDIR)/versions/VALDCLI_VERSION
+
+.PHONY: update/template
+## update PULL_REQUEST_TEMPLATE and ISSUE_TEMPLATE
+update/template:
+	$(eval TMPL_GO_VERSION      := $(shell cat $(ROOTDIR)/versions/GO_VERSION))
+	$(eval TMPL_KUBECTL_VERSION := $(shell cat $(ROOTDIR)/versions/KUBECTL_VERSION))
+	$(eval TMPL_NGT_VERSION     := $(shell cat $(ROOTDIR)/versions/NGT_VERSION))
+	sed -i -e "s/^- Go Version: .*$$/- Go Version: $(TMPL_GO_VERSION)/" $(ROOTDIR)/.github/ISSUE_TEMPLATE/bug_report.md
```

Comment:
```suggestion
	sed -i -e "s/^- Go Version: .*$$/- Go Version: $(@make version/go)/" $(ROOTDIR)/.github/ISSUE_TEMPLATE/bug_report.md
```

Reply:
Thank you for your suggestion!
I tried it, but an error occurred in my local environment 😢 
How about the following changes?? 🤔 
https://github.com/vdaas/vald/pull/2212/commits/8700c53cdad74974b15574967a0110e0d8b9c77b